_episodes/23-continuous-integration-automated-testing.md

```diff
@@ -370,7 +370,7 @@ jobs:
     strategy:
       matrix:
         os: ["ubuntu-latest", "macos-latest", "windows-latest"]
-        python-version: ["3.10", "3.11"]
+        python-version: ["3.10", "3.11", "3.12"]
```

Comment:
```suggestion
        python-version: ["3.10", "3.11"]
```

Reply:
Need to remove 3.12 as the text refers to 6 combinations and never mentions 3.12 in then text (but 3.10 and 3.11 are mentioned). Or add 3.12 but make sure all the text is updated as well.